pkg/discovery/dtofactory/pod_entity_dto_builder.go

```diff
@@ -385,6 +382,9 @@ func (builder *podEntityDTOBuilder) buyCommoditiesFromVolumes(pod *api.Pod, moun
 
 		provider := sdkbuilder.CreateProvider(proto.EntityDTO_VIRTUAL_VOLUME, providerVolUID)
 		dtoBuilder = dtoBuilder.Provider(provider)
+		dtoBuilder.IsMovable(proto.EntityDTO_VIRTUAL_VOLUME, false).
```

Comment:
Do you think we can just make it controllable false?

Reply:
I did replicate what other [probes are doing](https://rbcommons.com/s/VMTurbo/r/39397/diff/1/#0) wrt to the volumes as of now. I think we better stick to the storage teams recommendation.